packages/node/src/lib/subscribers/subscriber.interface.ts

```diff
@@ -8,7 +8,7 @@ import {
 export { ISubscriberPayload };
 
 export interface ISubscribers {
-  list(page: number);
+  list(page: number, limit: number);
```

Comment:
I think this change doesn't belong to this PR. Does it come from an unmerged one?

Reply:
Yes, I added this in subscriber doc PR but removed it later because it requires a new version to be released for `@node/novu` 
I will update the subscriber doc also after this PR merge